### DISCUSSION TITLE
Unable to acess Jar file

### DISCUSSION THREAD

**Discussion by HarshPatel5940** [jagrosh/MusicBot#915] [Questions] [answered]
Can Someone say why is it saying unable to access jar file.
I was using release 0.3.5

**Comment by YJSoft** [comment]
Without any information(environment like file path, OS, etc...) nobody can answer.

**Reply by HarshPatel5940** [reply]
path is the Repo path (not inside src/.../JmusicBot) its the full outer path.
OS is windows
java installed

**Reply by HarshPatel5940** [reply]
say me what all info u should provide so you can help me out

**Reply by HarshPatel5940** [reply]
I have created config.txt file and also filled the things in BotConfig.java 
![image](https://user-images.githubusercontent.com/83941233/134765304-25b58bf7-7b06-4551-ad69-34e978dd86af.png).


@YJSoft Umm Any Other things to provide for help??

**Comment by Sanduhr32** [comment, marked as answer]
Yes, in the directory, where you try to run `JMUSICBOT-0.3.5.JAR` needs to be the jar file.

Self compilation isn't supported. You seem to not be familiar with Java programming & executing any program through the command line. I strongly advise you to just download a release .JAR file, no zip or git cloning, and follow the setup guide provided in the wiki.

**Reply by HarshPatel5940** [reply]
yea i mostly into python bots so idk mush about java, bar or something....oh ok

thanks for the reply!!
Harsh Patel

**Reply by Sanduhr32** [reply]
Similarity to Python:
python3 main.py  for running your script(s).
java -jar MyJar.jar for running compiled java archives.

No need to thank me, if you strictly follow the guide you should get your MusicBot up and running.

If you have any issues, just let us know and we will help you.

**Reply by HarshPatel5940** [reply]
![image](https://user-images.githubusercontent.com/83941233/134766345-ec5e8f67-00a9-492a-9db3-d4be4bb02091.png)

the page is not allowing the connecttion, so I was just doing things blindly 😢. Anyways Nice Meeting you : )

**Reply by YJSoft** [reply]
@HarshPatel5940
https://github.com/jagrosh/MusicBot/blob/gh-pages/Installing-Java.md

https://github.com/jagrosh/MusicBot/blob/gh-pages/Setup.md

https://github.com/jagrosh/MusicBot/blob/gh-pages/Getting-a-Bot-Token.md

https://github.com/jagrosh/MusicBot/blob/gh-pages/Finding-Your-User-ID.md

Documentation source is github's gh-pages branch. If site is not accessible, try above link.

**Reply by Sanduhr32** [reply]
Oh well, you are chinese right? Well maybe your ISP or government blocks "unknown" pages and certain Top Level Domains.

**Reply by Sanduhr32** [reply]
I was just about to post:
https://github.com/jagrosh/MusicBot/blob/gh-pages/Setup.md

**Reply by HarshPatel5940** [reply]
@Sanduhr32 I am from India not from china lol

**Reply by Sanduhr32** [reply]
Sorry about that confusion from my side, but something blocks you from accessing the site. It works completely fine for me.

Do you have a spare VPN for testing if it's some good restriction?
